guides/common/images/foreman_iss_disconnected_airgapped.drawio

```diff
@@ -1 +1 @@
-<mxfile host="Electron" modified="2022-05-17T23:20:08.954Z" agent="5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) draw.io/15.3.8 Chrome/91.0.4472.164 Electron/13.5.0 Safari/537.36" etag="nynIQqBX6ozz9S7jC-y8" version="15.3.8" type="device"><diagram id="uq4QgXehMySjV9am80Ny" name="Page-1">7Vhdb9owFP01eWRK7PD1WEKh0sRWCY1Nfams2CUeThw5psB+/ZzGDvmifAxYtQE8xOfenMT33GMnWNAL12OB4mDCMWEWsPHagkMLAMcFwEp/Nt5kSM+GGTAXFOukLTClv4gGbY0uKSZJKVFyziSNy6DPo4j4soQhIfiqnPbCWfmqMZqTGjD1Eauj3ymWgUY7tr0NPBA6D/Sl+yYQIpOsgSRAmK8KELy3oCc4l9lRuPYIS4tn6pKdN9oRzW9MkEgecsLz55+z7uLH0+Bp6S3wpAt7wGlpllfElnrCHo9kypjds9yYQgi+jDBJuRwLDlYBlWQaIz+NrpT0CgtkyHT4RZFoLYFrxprLxD3OuHjjhjZoD3teilPGDB7xSKUPEin4gjQlZxGjiqrnoF4SMz8iJFkXIF2iMeEhkWKjUnQUGrl0v7p6uCqIb1KCgu4mD+l+m+fMW0nUgVblCIVATaERFyREkQKnRKiZ/a9SqbWkJJUyZU0rYF9VK7cmVk0cEuG7dF1SI5+hJKH+Tj0c9536n6XYCV8Kn+xfHyQScyL3dynBpdW0Ll1BmnaDMgYThCFJX8trcJNc+gqPnKZLVm5it9wZsKp4Nm99VnHNrBL1y0SgXyHKClMjeuuefNp/0FDtf62hPlajuM4efQ9tFLe6FvUu1ijQWcxm3/o9nHxt+9GXcLIYP7TgbZc4cJfIN/hr7BKNWjn9o0yty3ayo4v1xCgJcsnTwSOSkojoDQE2aJKn76VfFVH3NKLpVIf2e9J8FGtXDAlOdHYX7CG6sLNB4yN6+q6jhNztafsoTzudc3q6BJ7ZzhVZe1d8PG+Wp/58fjPzpc2saD/ZxQ88j7f38V7a6vVd/I6K1hjF8c3rhV3675n9uPe7m9nPYvZuxZSdM5l9D+/JZlfD7b96Wfr2v1F4/xs=</diagram></mxfile>
\ No newline at end of file
+<mxfile host="app.diagrams.net" modified="2021-10-08T11:43:36.193Z" agent="5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.0 Safari/537.36" etag="AViib8Li84wyJbw4Blof" version="15.2.9" type="device"><diagram id="uq4QgXehMySjV9am80Ny" name="Page-1">7Vhdk5owFP01PG4HElR4rLi6Mx3bnXFqO/uyk4GspAJxQly1v77JkiBfFrVoO1v1QXLu5UDuuScBDejF2wlDq3BKAxwZwAy2BhwZAFimPRA/EtllSL9vZ8CCkUAl7YEZ+Yn1mQpdkwCnpUROacTJqgz6NEmwz0sYYoxuymkvNCpfdYUWuAbMfBTV0W8k4KFC+6a5DzxgsgjVpV0diJFOVkAaooBuChC8N6DHKOXZUbz1cCSLp+uSnTc+EM1vjOGEH3PC86cf88Hy+9Pwae0tg+kAOsC6UyyvKFqrCXs04ZIxu2e+04VgdJ0EWHJZBhxuQsLxbIV8Gd0I6QUW8jhS4RdBorQEth4rLh33aETZGzc0QW/keBInUaTxhCYifZhyRpe4KTmLaFVEPYf1kuj5YcbxtgCpEk0wjTFnO5GiolDLpfrVVsNNQXydEhZ013lI9dsiZ95LIg6UKicoBGoKjSnDMUoEOMNMzOx/lQo4ZamEKWtaAfOqWtk1sWri4CT4KNclMfIjlKbEP6iHZf+m/p0UO6Vr5uP29YEjtsC8vUtxUFpN69IVpOk1KKMxhiPEyWt5DW6SS13hkRK5ZOUmtsudAauKZ/NWZxXXzCqRWyYCboUoK0yN6K178mn/QUP13ltD/VuNYlst+h7bKHZ1LXIu1ijQWs7nX10nSL/0/ORzPF1OHu7ge94laltCQ+sdvUvkG/w1dolGrSz3JFOrsp3t6GI9A5SGueRy8Ig4xyx5Q4AJmuRxPfkVEXFPYyKnOjJzaSo+PkqYK1m7YkhwprMHoIXows4GjY/o8l1HCHnY0+ZJnrb6XXq6BHZs54qszhUfz5vlqT+f38x8aTML2g9m8QO78XYb76WtXt/FRyT1b27P9TH/ut1Pe8O72b0Tuw8qtux3ZPcW3rPtLob7//Wy9P2/o/D+Fw==</diagram></mxfile>
```

Comment:
General question: Does this mean we're dropping "air-gapped" in favor of disconnected?

Reply:
Oops, no, it doesn't. I must have removed it by mistake. Thanks for the catch!

Reply:
We definitely need better guidance on the connected/disconnected/airgapped scenarios in the docs. We're planning to address it in 3.3. I'll file an issue for community discussion soonish.

Reply:
OK, but you _have_ changed it, that's why I'm asking:

<img width="1080" alt="Screenshot 2022-07-22 at 22 06 38" src="https://user-images.githubusercontent.com/12595287/180519171-ef11ec2f-8db3-437e-85f2-781652b17316.png">

Reply:
Ohhh, I see. That's a mistake. Thanks for catching that!

Reply:
Corrected.